pkg/imager/profile/testdata/akamai-amd64-1.10.0.yaml

```diff
@@ -0,0 +1,26 @@
+arch: amd64
+platform: akamai
+secureboot: false
+version: 1.10.0
+customization:
+  metaContents:
+    - key: 17
+      value: dual-boot
+input:
+  kernel:
+    path: /usr/install/amd64/vmlinuz
+  initramfs:
+    path: /usr/install/amd64/initramfs.xz
+  sdStub:
+    path: /usr/install/amd64/systemd-stub.efi
+  sdBoot:
+    path: /usr/install/amd64/systemd-boot.efi
+  baseInstaller:
+    imageRef: ghcr.io/siderolabs/installer:1.10.0
+output:
+  kind: image
+  imageOptions:
+    diskSize: 2356150272
+    diskFormat: raw
+    bootloader: dual-boot
```

Comment:
maybe this should be a quirk?

Reply:
these are test stubs, not the actual code

Reply:
ah okay, didn't see it's testdata